CITATION.cff

```diff
@@ -4,15 +4,18 @@ message: |
   Authors are listed in alphabetical order.
 
 title: "earthaccess"
-doi: "https://doi.org/10.5281/zenodo.8365009"
+doi: "10.5281/zenodo.8365009"
 abstract: "Python Library for NASA Earthdata APIs"
 contact:
   - name: "The earthaccess community"
     website: "https://github.com/nsidc/earthaccess/discussions"
   - name: "NSIDC"
     email: "nsidc@nsidc.org"
 type: "software"
-license: ["MIT"]
+# NOTE: The CFF spec says `license` can be a list, but Zenodo is currently not
+#       accepting lists for this key:
+#       https://github.com/zenodo/zenodo/issues/2515
+license: "MIT"
```

Comment:
This was the thing that broke everything. All the other changes are from adding the validator.